math/wnaf.go

```diff
@@ -3,24 +3,33 @@ package math
 
 import "math/big"
 
-// SignedDigit obtains the signed-digit recoding of n such that n>0 and odd and
-// returns a list L such that n = sum( L[i]*2^(w-1) ) and L[i] are odd numbers
-// in the set {±1, ±3, ..., ±2^(w-1)-1}, where 1 < w < 32.
+// SignedDigit obtains the signed-digit recoding of n and returns a list L of
+// digits such that n = sum( L[i]*2^(w-1) ), and each L[i] is an odd number
```

Comment:
I think i belongs in the exponent somewhere.

Reply:
good catch!